src/2XXXR0_a_motivation_and_plan_for_a_physical_quantities_and_units_library.md

```diff
@@ -513,7 +515,7 @@ _Note:_ No built-in support for text output of `quantity_point`.
 
 | Feature                           | Priority | Description                                                                                                                                                                                                                      |
 |-----------------------------------|:--------:|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| The most important ISQ quantities |    1     | Specification of the most commonly used ISQ quantities                                                                                                                                                                           |
+| The most important ISQ quantities |    1     | Specification of the most commonly used ISQ quantities <!-- Which are those? -->                                                                                                                                                 |
```

Comment:
The ones you can find in all other libraries on the market (i.e. Boost.Units). Those would be enough to implement SI units and solve common cases. We will have to hand-pick them if needed in an arbitrary way.

This point is a gateway for people scared of standardizing hundreds of ISQ quantities.